### PR TITLE
add from to logger

### PR DESCRIPTION
**Pull request by AlvaroVega**

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,2 +1,3 @@
+- Add `from` based on fowarder header in log context
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]: Similar question to the one in IOTAs repos... is there any document about logs in this repo that should be modified based on this PR?

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE:1]: I've found this doc https://github.com/telefonicaid/fiware-pep-steelskin/blob/b766e6c2013422195d66cf07c5ce320c85d0ca49/operations.md#--logs but no format is described

Comment (fgalan) [reply CHANGES_NEXT_RELEASE:1]: NTC

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**